src/routes/user.js

```diff
@@ -10,6 +15,9 @@ const { validateEmail } = EmailValidation;
 const { verifyUser, inputNewPassword, setNewPassword } = UserController;
 
 const { resendConfirmationEmail, sendResetPasswordEmail } = EmailController;
+// const { verifyAuth } = profileMiddleware;
+// const { resendConfirmationEmail, verifyUser } = UserController;
```

Comment:
@Procech20  why are these commented out???

Reply:
They are no longer in use anywhere